bee-storage/bee-storage-test/src/message_id_to_message.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::{Message, MessageId};
+use bee_storage::{
+    access::{AsIterator, Batch, BatchBuilder, Delete, Exist, Fetch, Insert, MultiFetch, Truncate},
+    backend,
+};
+// use bee_test::rand::message::{rand_message, rand_message_id};
+
+pub trait StorageBackend:
+    backend::StorageBackend
+    + Exist<MessageId, Message>
+    + Fetch<MessageId, Message>
+    + for<'a> MultiFetch<'a, MessageId, Message>
+    + Insert<MessageId, Message>
+    + Delete<MessageId, Message>
+    + BatchBuilder
+    + Batch<MessageId, Message>
+    + for<'a> AsIterator<'a, MessageId, Message>
+    + Truncate<MessageId, Message>
+{
+}
+
+impl<T> StorageBackend for T where
+    T: backend::StorageBackend
+        + Exist<MessageId, Message>
+        + Fetch<MessageId, Message>
+        + for<'a> MultiFetch<'a, MessageId, Message>
+        + Insert<MessageId, Message>
+        + Delete<MessageId, Message>
+        + BatchBuilder
+        + Batch<MessageId, Message>
+        + for<'a> AsIterator<'a, MessageId, Message>
+        + Truncate<MessageId, Message>
+{
+}
+
+pub fn message_id_to_message_access<B: StorageBackend>(_storage: &B) {
```

Comment:
Is this commented beacuse of the `bee_test` crate? These should be coming pretty soon, working on them in the background.

Reply:
Yes, because of bee-test. I'll do a second PR to include the storage tests properly.